openedxscorm/scormxblock.py

```diff
@@ -42,6 +42,7 @@ def _(text):
 
 
 logger = logging.getLogger(__name__)
+os.path.altsep = '\\'
```

Comment:
Can you define this as constant instead of overriding python library's internal property?

Reply:
Updated.

Reply:
Updated to `OS_PATH_ALT_SEP`.